config/registry.yaml

```diff
@@ -308,6 +312,17 @@ modules:
       api_key: optional
       timeout: optional
       values: optional
+  byo-cert:
```

Comment:
Let's name this "ssl-certificate"? `byo-cert` is not clear enough

Reply:
I thought ssl-certificate was not clear as it would imply we would create it for them lol. How about "external-ssl-cert"?

Reply:
`external-ssl-cert` sounds good!